core/src/test/scala/unit/kafka/network/SocketServerTest.scala

```diff
@@ -818,7 +830,7 @@ class SocketServerTest extends JUnitSuite {
     val kafkaMetricNames = metrics.metrics.keySet.asScala.filter(_.tags.asScala.get("listener").nonEmpty)
     assertFalse(kafkaMetricNames.isEmpty)
 
-    val expectedListeners = Set("PLAINTEXT", "TRACE")
+    val expectedListeners = Set("PLAINTEXT", "TRACE", "CONTROLLER")
```

Comment:
Hmm, the default config doesn't seem to specify a control listener, why do we expect the controller listener?

Reply:
You are right, its not required.